packages/react-sdk/src/remix.ts

```diff
@@ -0,0 +1,47 @@
+const getRemixSegment = (segment: string) => {
+  if (segment === "*") {
+    return "$";
+  }
+  const match = segment.match(/^:(?<name>\w+)(?<modifier>\*|\?)?$/);
+  const name = match?.groups?.name;
+  const modifier = match?.groups?.modifier;
+  if (name) {
+    if (modifier === "*") {
+      return "$";
+    }
+    if (modifier === "?") {
+      return `($${name})`;
+    }
+    return `$${name}`;
+  }
+  return `[${segment}]`;
+};
+
+export const getRemixRoute = (pathname: string) => {
```

Comment:
Plz use `/**` comments to give a short description from what format to wht format you are converting with links etc on originnal docs

Reply:
i.e.
```ts
/**
 * Transformrs [URL ROUTE](link) format to [Remix](file-route-link) format
**/
export const getRemixRoute = (pathname: string) => {
```